packages/vue/src/components/molecules/SfCallToAction/SfCallToAction.stories.js

```diff
@@ -1,27 +1,11 @@
 import { storiesOf } from "@storybook/vue";
-import {
-  withKnobs,
-  text,
-  color,
-  optionsKnob as options
-} from "@storybook/addon-knobs";
+import { withKnobs, text, color } from "@storybook/addon-knobs";
 import SfCallToAction from "./SfCallToAction.vue";
 storiesOf("Molecules|CallToAction", module)
   .addDecorator(withKnobs)
   .add("Common", () => ({
     components: { SfCallToAction },
     props: {
-      customClass: {
```

Comment:
This cause an error in console - please remove the `customClass` prop assignment in the stories template :)